com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -54,35 +54,40 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
 
 ## Materials parameterization
 
-Some phenomena like refraction, absorption in transmissive objects, or subsurface scattering, can be expressed more naturally in a path tracing setting than in its rasterized counterpart, and as such require less material parameters (e.g. additional thickness information, expected shape of the refractive object, ...). For that reason, some parameters have no effect in path tracing, while others bear a slightly different meaning.
+HDRP can perform effects like light refraction and absorption in transmissive objects, or subsurface scattering, more easily when you use path tracing setting than in rasterization . This means they need fewer Material parameters (e.g. additional thickness information, expected shape of the refractive object, ...). For that reason, some parameters have no effect in path tracing, and some parameters change behaviour. For more information, see [Refraction models](#).
```

Comment:
It's not performing those effects "more easily" (as in, it's not computing them faster, or with less resources), but rather can express them "more naturally" given the different way of representing light transport simulation.

Reply:
> Thanks for your feedback @eturquin I'm implementing your suggested changes in a[ google doc](https://docs.google.com/document/d/1dC2syNIT1hGV-Bz64RGClzdZfXbUVhx0sSiKZ9eQYUg/edit) so I can get it reviewed by docs editorial.

Sounds good! :)